src/utils/ethereum.ts

```diff
@@ -34,32 +48,36 @@ async function getTokens(wallet, hermezTokens) {
         type: "function",
       },
     ];
-    const balancePromises = hermezTokens.map((token) => {
+    const balancePromises: Promise<BigNumber>[] = hermezTokens.map((token) => {
       if (token.id === ETHER_TOKEN_ID) {
         // tokenID 0 is for Ether
         return provider.getBalance(ethereumAddress);
       } else {
         // For ERC 20 tokens, check the balance from the smart contract
         const contract = new ethers.Contract(token.ethereumAddress, partialERC20ABI, provider);
 
-        return (
-          contract
-            .balanceOf(ethereumAddress)
-            // We can ignore if a call to the contract of a specific token fails.
-            .catch(() => {})
-        );
+        // We can ignore if a call to the contract of a specific token fails.
+        // ToDo: Find a way to properly type the functions of the contract declared through the ABI.
+        // eslint-disable-next-line
+        const balance: Promise<BigNumber> = contract
+          .balanceOf(ethereumAddress)
+          // eslint-disable-next-line @typescript-eslint/no-empty-function
+          .catch(() => {});
```

Comment:
To prevent disabling the `eslint` I think that you can do: `.catch(() => ({}));`

Reply:
Nice trick! All they are now removed. Thx!